tasks.py

```diff
@@ -283,7 +283,7 @@ def lock(context, check=False, constrain_nautobot_ver=False, constrain_python_ve
         except UnexpectedExit:
             print("Unable to add Nautobot dependency with version constraint, falling back to git branch.")
             command = (
-                f"poetry add --lock git+https://github.com/nautobot/nautobot.git#{context.nautobot_ssot.nautobot_ver}"
+                f"poetry add --lock git+https://github.com/nautobot/nautobot.git#v{context.nautobot_ssot.nautobot_ver}"
```

Comment:
Just want to call out that this is not meant to work for the tagged versions such as `v2.0.0` instead this was done to allow installing from `develop` or `next` which do not start with the `v`. I think we should add more details on the error message as the error is very misleading.

Reply:
This was done just as a test to see if it'd at least get me a bit further.